packages/eas-cli/README.md

```diff
@@ -54,7 +52,7 @@ USAGE
   $ eas build
 ```
 
-_See code: [src/commands/build/index.ts](https://github.com/expo/eas-cli/blob/v0.0.0/src/commands/build/index.ts)_
+_See code: [build/commands/build/index.ts](https://github.com/expo/eas-cli/blob/v0.0.0/build/commands/build/index.ts)_
```

Comment:
Why are the .ts files under `build`?

Reply:
These lines were automatically added, by the `oclif-dev readme` command, not sure why it included the paths from build output. Re-running it seems to have removed them.